src/Form.js

```diff
@@ -0,0 +1,31 @@
+
+const Form =({selectCuisine, handleSubmit}) => {
+
+  return (
+  <form action="" onSubmit={ (event) => {handleSubmit(event)}}>
+	<label>Find a recipe:</label>
+	<select 
+		name="" 
+    id="" 
+    onChange={(event) => {
+			selectCuisine(event.target.value)
+		}}
+		>
+		<option value="placeholder" disabled>Select cuisine</option>
```

Comment:
Inside your option tag on line 14 you can use _value=**""**_ in order for the option Select Cuisine to show to the user.